sdk/core/System.Net.ClientModel/tests/client/OpenAIClient/Choice.Serialization.cs

```diff
@@ -13,8 +15,11 @@ namespace OpenAI;
 
 public partial class Choice
     {
-        internal static Choice DeserializeChoice(JsonElement element)
+        internal static Choice DeserializeChoice(JsonElement element, ModelReaderWriterOptions options)
         {
+            bool wire = options.Format == ModelReaderWriterFormat.Wire;
```

Comment:
Do we need to push this pattern through to all the types in this test client?  

If you don't want to do that as part of this PR, @KrzysztofCwalina, it would be helpful to file an issue to make sure the work is tracked.

Reply:
Ultimately, we will be generating the serialization code and so I am not sure we want to even file an issue. Let me know if you still want to and I will file.

Reply:
That's fine.  My concern is primarily that people new to the feature branch won't know which hand-implementations are a source of truth for our patterns when copying/referencing, but I think we can contain that for now.  With the addition of the new [MapsClient](https://github.com/Azure/azure-sdk-for-net/tree/feature/core-experiment/sdk/core/System.Net.ClientModel/tests/client/MapsClient) test client, we already have some pattern divergence so will need to keep up with this in our heads until we sit down and make a concerted effort to unify.